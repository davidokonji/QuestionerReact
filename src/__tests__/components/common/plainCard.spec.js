import { shallow } from 'enzyme';
import React from 'react';
import PlainCard from '../../../components/common/plainCard';

describe('PlainCard component', () => {
  it('should render correctly', () => {
    const history = {
      push: jest.fn()
    };
    const wrapper = shallow(<PlainCard history={history} />);

    wrapper.find('.plaincard').simulate('click');
    expect(wrapper).toMatchSnapshot();
  });
});
