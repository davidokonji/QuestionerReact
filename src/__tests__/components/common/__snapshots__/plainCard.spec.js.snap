// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlainCard component should render correctly 1`] = `
<div
  className="col-12 col-md-4 mt-3 plaincard"
  onClick={[Function]}
  onKeyDown={null}
  role="button"
  tabIndex="-1"
>
  <div
    className="card shadow-sm"
  >
    <div
      className="card-body col-12"
    >
      <h3
        className="mt-0"
      />
      <p
        className="mt-0"
      />
      <p
        className="mt-0"
      />
    </div>
  </div>
</div>
`;
